index.html

```diff
@@ -28,5 +28,19 @@
         </div>
     </nav>
 
+    <section class="headline">
```

Comment:
[OPTIONAL] Kindly make sure your design matches with the Figma template. Currently, there are some problems with nav and headline. Please try to give a display flex instead of a fixed value. 
<img width="635" alt="Screenshot 2023-03-24 at 10 09 44 PM" src="https://user-images.githubusercontent.com/25887253/227623340-7904ac37-ebdd-4125-9ce8-1c77fea2fabb.png">

<img width="452" alt="Screenshot 2023-03-24 at 10 37 55 PM" src="https://user-images.githubusercontent.com/25887253/227623326-4cea5d2a-570b-4441-8898-4ae874228be7.png">

Reply:
Hello, thanks for your review. In my browser this problem does not happen. Any idea why that could be? the toolbar width is set to 100% so there shouldn't be any empty space.

![image](https://user-images.githubusercontent.com/117373513/227629514-ab7ccd06-3473-4f17-b101-749bfb6a35e7.png)

Reply:
Hey @tomasesquivelgc. I think It needs to be a 100%.
<img width="698" alt="Screenshot 2023-03-25 at 12 17 53 AM" src="https://user-images.githubusercontent.com/25887253/227643697-b1156877-3bd4-41fb-8af4-d4abc1e2618c.png">